### PR TITLE
Rjschad recommends

### PR DESCRIPTION
**Pull request by rjschad**
need to include more information

### CODE REVIEW

**Comment by alecaldapa** [comment]
Looks great! Keep up the good work.

File: rjschad.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Hometown: Ridgecrest, CA
+
+
+### Places to eat
+
+- Aviator's Smokehouse
+- Cooleys
+
```
Comment (rjschad) [inline rjschad.md]: In-n-out

File: rjschad.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### Hometown: Ridgecrest, CA
+
+
+### Places to eat
+
+- Aviator's Smokehouse
+- Cooleys
+- In-n-out
+
```
Comment (SUK109) [inline rjschad.md:10]: What is your favorite burger?

**Comment by rjschad** [comment]
Double-Double with animal style fries ;)

**Pull request merged**